bot.py

```diff
@@ -0,0 +1,27 @@
+import discord
```

Comment:
Why are u making changes to bot.py?

Reply:
kindly check...i have removed bot.py and have only pushed pip file.And if i must do anything else please tell